connection.js

```diff
@@ -239,10 +238,16 @@ class Connection {
         }
     }
     set (obj, prop, val) {
+        const self = this;
```

Comment:
This seems like a pointless addition. Other than that this change looks fine.

Reply:
*nod* - changed.